src/components/Dropdown/FilterDropdown/FilterDropdown.tsx

```diff
@@ -93,25 +99,58 @@ export const FilterDropdown: FC<Props & ElementProps> = ({
         <DropdownScrollArea>
           <ContentLayout themes={themes}>{children}</ContentLayout>
         </DropdownScrollArea>
-        <BottomLayout themes={themes}>
-          {onReset && (
-            <ResetButtonLayout themes={themes}>
-              <Button variant="text" size="s" prefix={<FaUndoAltIcon />} onClick={onReset}>
-                {resetButton}
-              </Button>
-            </ResetButtonLayout>
+        <ActionArea themes={themes}>
+          <Cluster gap={1} align={'center'} justify={'space-between'}>
```

Comment:
imo： 特に問題ないですが、ここだけ書き方が違って気になりました。

```suggestion
          <Cluster gap={1} align="center" justify="space-between">
```

Reply:
@uknmr コメントありがとうございます。修正しました！https://github.com/kufu/smarthr-ui/pull/3348/commits/a0ac3ae5544ede0fddf4d87b2a27ec91f22a18f1